test/undo.t

```diff
@@ -560,6 +560,53 @@ class TestUndo(TestCase):
                                   expectedEnd=one_hour_before_utc,
                                   expectedTags=["bar", "foo"])
 
+    def test_undo_entries_config_one(self):
```

Comment:
> Test description and test function name should be more specific...

Hopefully the latest push is more clear (but still contained to a single line)

> What is the purpose of this/these tests?
> If a journal is set to a certain size `n`, the user should be able to undo `n` steps.
> I.e. the test should set the journal size to `n`, apply `n+1` steps and verify that for the `(n+1)th` call to the undo command "nothing to undo" is reported (should this be an error?)

Yes, but are you saying you would like to see this description in python comment string for the method? 

> Another test would be, that if the journal file has `n` lines and the test sets `journal.size` to `n`, the file does not grow after a command call. Or if the size has `m<n` lines, the file grows only to size `n`.

This is a little tricky since the file size can change without there being new entries depending on the contents of each transaction. I also took your advice and didn't want to look into the file or parse any output from the undo command due to the brittleness of doing so. That is why the test only uses the result of the export command to see if the database has changed.

> This also means we cannot test for an infinite journal...

/me nods